docs/tutorials/image_classification/kaggle.md

```diff
@@ -15,7 +15,7 @@ After completing this step, you will have the following directory structure on y
     ├── ...
 ```
 
-Here `data` is a folder containing the raw images categorized into classes. For example, subfolder `class1` contains all images that belong to the first class, `class2` contains all images belonging to the second class, etc. 
+Here `data` is a folder containing the raw images categorized into classes. For example, subfolder `class1` contains all images that belong to the first class, `class2` contains all images belonging to the second class, etc.
```

Comment:
Where is the kaggle tutorial in the doc build? I can't find it.

Reply:
http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-733/56/tutorials/image_classification/kaggle.html